cvat/apps/dataset_manager/annotation.py

```diff
@@ -472,7 +472,7 @@ def simple_interpolation(shape0, shape1):
             for frame in range(shape0["frame"] + 1, shape1["frame"]):
                 offset = (frame - shape0["frame"]) / distance
                 points = None
-                if shape1["outside"]:
+                if shape0["outside"] and shape1["outside"]:
```

Comment:
@kirill-sizov , could you please explain the change?

Reply:
I made this change so that in the case where the shape has the values `keyfame = True` and `outside = True`, the interpolation still occurs as shown below:
![3300](https://user-images.githubusercontent.com/43179655/122811348-09dac300-d2d9-11eb-948e-efe7c1b06cfb.gif)
In current develop version this only works in the UI, but when exporting annotations, there is no interpolation between such frames.

Reply:
@kirill-sizov ,we have 4 cases:
- both frames are inside => usual interpolation
- first frame is inside, second is outside => usual interpolatio
- first frame is outside, second frame is inside => need to return 2 shapes (first, second), no need to interpolate
- first frame is outside, second frame is outside => need to return 2 shapes (first, second), no need to interpolate.

Could you please implement and check if it works as expected?

Reply:
Yes sure, I'll do it.